src/react-components/ui-root.js

```diff
@@ -407,6 +422,14 @@ class UIRoot extends Component {
               subtitle={this.state.availableVREntryTypes.daydream == VR_DEVICE_AVAILABILITY.maybe ? daydreamMaybeSubtitle : "" }/> }
         { this.state.availableVREntryTypes.cardboard !== VR_DEVICE_AVAILABILITY.no &&
           (<div className="entry-panel__secondary" onClick={this.enterVR}><FormattedMessage id="entry.cardboard"/></div>) }
+        { !mobiledetect.mobile() && /firefox/i.test(navigator.userAgent) && (
```

Comment:
this firefox test surprised me -- perhaps extract it into a variable to explain why its needed?